.github/workflows/run_rc_validation.yml

```diff
@@ -0,0 +1,533 @@
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.m
+
+name: Run RC Validation 
+on:
+  workflow_dispatch:
+    inputs:
+      RELEASE_VER:
+        description: Beam current Release Version
+        required: true
+        default: 2.42.0 
+      USER_GCS_BUCKET:
+        description: Bucket to upload results
+        required: true
+        default: gs://rc-validation-migration-tests
+      RC_NUM:
+        description: RC number
+        required: true
+        default: "1"
+      APACHE_CONTENTS_REPO:
+        description: Apache Contents Repository
+        required: true
+        default: https://repository.apache.org/content/repositories/orgapachebeam-1285
+      RUN_SQL_TAXI_WITH_DATAFLOW:
+        description: Include SQL Taxi with Dataflow?
+        required: true
+        type: boolean
+        default: true
+      RUN_PYTHON_CROSS_VALIDATION:
+        description: Include Python Cross Validation?
+        required: true
+        type: boolean
+        default: true
+      RUN_DIRECT_RUNNER_TESTS:
+        description: Include Direct Runner Leaderboard & Gamestates?
+        required: true
+        type: boolean
+        default: true
+      RUN_DATAFLOW_RUNNER_TESTS:
+        description: Include Dataflow Runner Leaderboard & Gamestates?
+        type: boolean
+        required: true
+        default: true
+env: 
+  RC_TAG: "v${{github.event.inputs.RELEASE_VER}}-RC${{github.event.inputs.RC_NUM}}"
+  RELEASE_VER: ${{github.event.inputs.RELEASE_VER}}    
+  USER_GCP_PROJECT: apache-beam-testing
+  PYTHON_RC_DOWNLOAD_URL: https://dist.apache.org/repos/dist/dev/beam
+  USER_GCP_REGION: us-central1
+  USER_GCP_ZONE: us-central1-a
+  APACHE_CONTENTS_REPO: ${{github.event.inputs.APACHE_CONTENTS_REPO}}
+  FIXED_WINDOW_DURATION: 20
+jobs:
+  python_release_candidate:
+    runs-on: [self-hosted, ubuntu-20.04]
+    permissions:
+      pull-requests: write
+      contents: write
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      RELEASE_BRANCH: "release-${{github.event.inputs.RELEASE_VER}}"
+      WORKING_BRANCH: "v${{github.event.inputs.RELEASE_VER}}-RC${{github.event.inputs.RC_NUM}}_validations"
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ env.RC_TAG }}
+      - name: Setup GitHub CLI
+        uses: ./.github/actions/setup-gh-cli-linux
+      - name: Set git config
+        run: |
+          git config user.name $GITHUB_ACTOR
+          git config user.email actions@"$RUNNER_NAME".local
+      - name: Create Pull Request
+        run: |
+          git checkout -b ${{env.WORKING_BRANCH}} ${{ env.RC_TAG }} --quiet
```

Comment:
Can we do some validation to ensure this branch doesn't already exist (theoretically it shouldn't, but that could break the validation)